npe2/manifest/schema.py

```diff
@@ -68,10 +68,14 @@ class PluginManifest(BaseModel):
         None,
         description="The publisher name - can be an individual or an organization",
     )
-    # easy one... we need this.  character limit?  256 char?
+
     display_name: str = Field(
         "",
         description="The display name for the extension used in the Marketplace.",
```

Comment:
lets' change the word marketplace while we're at it

Reply:
ha you got it.